tox.ini

```diff
@@ -0,0 +1,21 @@
+[tox]
+envlist = py{27,35}-django{18,19}, quality
+
+[testenv]
+setenv =
+    # This allows us to reference test_settings.py
+    PYTHONPATH = {toxinidir}
+
+deps =
+    django18: Django>=1.8,<1.9
+    django19: Django>=1.9,<1.10
+    -rtest_requirements.txt
+
+commands =
+    django-admin.py test edx_rest_framework_extensions --settings=test_settings --with-coverage --cover-package=edx_rest_framework_extensions
+    coverage report
```

Comment:
+1. FWIW: When I've done this, I added a separate `coverage` testenv.
From there, the `coverage` testenv would run by default locally, but then I would instead run the  `coveralls` target on Travis (where the coveralls target both calculates coverage and also submits it to coveralls.io .